src/s1/cmp.js

```diff
@@ -0,0 +1,135 @@
+// __cmp('setConsentUiCallback', callback) QUANTCAST
+import 'core-js/fn/array/find-index';
```

Comment:
Using this library in one of our sites will double up the bundle size for anything we're using babel-polyfill for. I think what we want to do is something like this:
https://stackoverflow.com/questions/33380063/what-is-the-best-way-to-include-babel-polyfill-using-multiple-entry-points

In `development` mode in webpack we'd include the polyfill but in production mode we wouldn't include it

Reply:
Another reference is here:
https://babeljs.io/docs/en/babel-polyfill.html#usage-in-node-browserify-webpack

Reply:
Im not using babel-polyfill because it added another 130k to the package. But, would be nice to use it, ill make a ticket (see test plan)